bee-tangle/src/vertices.rs

```diff
@@ -76,24 +76,32 @@ impl Vertices {
         .ok()
     }
 
-    pub(crate) async fn pop_random(&self) -> Option<Vertex> {
-        let index = rand::thread_rng().gen_range(0..self.tables.len());
-        // SAFETY: `index < self.tables.len()` by construction.
-        let mut table = unsafe { self.tables.get_unchecked(index) }.write().await;
+    pub(crate) async fn pop_random<const RETRIES: usize>(&self) -> Option<Vertex> {
```

Comment:
What's the advantage of having RETRIES a generic param ?

Reply:
to be honest it could be a regular function parameter. I just wanted to be able to control the number of retries from the `Tangle` instead.

Reply:
Was thinking that maybe this is the kind of things we may want to have configurable to be able to tweak without recompiling

Reply:
Ok I'll add it to the configuration then